packages/sitecore-jss-react/src/components/BYOCComponent.test.tsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import { expect } from 'chai';
+import { mount } from 'enzyme';
+import { BYOCComponent } from './BYOCComponent';
+import { MissingComponent, MissingComponentProps } from './MissingComponent';
+
+describe('BYOCComponent', () => {
+  it('should render ExternalComponent with props when ComponentProps is provided', () => {
```

Comment:
Does ExternalComponent support SSR? If so - should we see the BYOC component rendered too?

Reply:
aah, i don't think it supports SSR. Maybe I should change the description of the test case.

Reply:
Verify with Yarik if it does -  we actually want SSR here.

Reply:
Pretty sure SSR wouldn't be working just yet. We have an upcoming task to do this for BYOC (JSS-491).